LabExT/Movement/PathPlanning.py

```diff
@@ -186,7 +186,8 @@ class PotentialField:
     def __init__(
         self,
         calibration: Type["Calibration"],
-        target_position: Type[ChipCoordinate],
+        target_coordinate: Type[ChipCoordinate],
+        start_z_lift: float = None,
```

Comment:
Get rid of this parameter. `PathPlanning` should always just stay on the chip z-coordinate of each starting point, i.e. should always move on the fixed xy-plane on a fixed z-level.

All the lifting/lowering should be done elsewhere, i.e. `move_absolute`.